.eslintrc.json

```diff
@@ -31,13 +45,42 @@
   },
   "overrides": [
     {
-      "files": ["**/__tests__/*.{j,t}s?(x)", "**/tests/unit/**/*.spec.{j,t}s?(x)"],
+      "files": [
+        "**/tests/*.{j,t}s?(x)",
+        "**/tests/unit/**/*.{j,t}s?(x)"
+      ],
       "env": {
         "jest": true
       },
+      "globals": {
+        "badRequest": true,
+        "cardService": true,
+        "io": true,
+        "ready1": true,
+        "request": true,
+        "socket1": true,
+        "socket2": true,
+        "socket3": true
+      },
+      "rules": {
+        "import/no-unresolved": [
+          "off"
+        ]
+      }
+    },
+    {
+      "files": [
+        "**/tests/e2e/**/*.{j,t}s?(x)"
+      ],
```

Comment:
Also added a second entry for the e2e folder which would needed cypress specific additions (`env` and `plugin` which already exists above).